deploy/Dockerfile.ppc64le

```diff
@@ -1,5 +1,7 @@
 FROM ppc64le/alpine:3.15
 MAINTAINER dashpole@google.com lysannef@us.ibm.com
+# Deprecated: the Dockerfile in this directory should support ppc64le
```

Comment:
We should be able to remove this file now. I added the deprecation message to be cautious about breaking people's workflow.

Reply:
ack, sounds good!